packages/terra-form-select/src/multiple/Frame.jsx

```diff
@@ -354,7 +359,7 @@ class Frame extends React.Component {
     // Don't blur if we dismissed the onscreen keyboard
     // Determined by if we have have interacted with the frame via onTouchStart
     // and if the focus is on input.
-    if (focusedByTouch && (relatedTarget === this.input)) {
+    if (relatedTarget === this.input) {
```

Comment:
here as well I assume `focusedByTouch` was added specifically added to prevent Blur for touch-devices. bit confused how are we handling now this scenario by removing `focusedByTouch`?

Reply:
Same as [above](https://github.com/cerner/terra-core/pull/3082#discussion_r458882200)